src/main/java/com/google/devtools/build/remote/client/LogParserUtils.java

```diff
@@ -236,6 +249,27 @@ private void printEntriesInOrder(OutputStream outStream) throws IOException, Par
     }
   }
 
+  /**
+   * Prints each entry out individually (ungrouped) and a message at the end for how many entries
+   * were printed/skipped.
+   */
+  private void printEntriesInJson() throws IOException, ParamException {
+    try (InputStream in = openGrpcFileInputStream()) {
```

Comment:
Likewise, this should be a stream transform.

Reply:
Sure, thanks